### PR TITLE
feat(map): extends for Map Type

### PR DESCRIPTION
**Pull request by Jonathan-Casey**
### Changes

Extends vs-code extension for Map Type syntax highlighting

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The extention adds code snippets for the following elements of the Concerto lang
 | Transaction | transaction |
 | Concept     |   concept   |
 | Enum        |    enum     |
+| Map        	|    map     	|
```
Comment (DS-AdamMilazzo) [inline README.md:59]: Indentation is off.

File: test-data/hr/hr.cto [hunk 2]
```diff
@@ -20,6 +25,7 @@ concept Address {
 concept Company {
     o String name
     o Address headquarters
+		o Rolodex rolodex
```
Comment (DS-AdamMilazzo) [inline test-data/hr/hr.cto:28]: Ditto about indentation, also above in Rolodex.

**Review by mttrbrts — APPROVED** [review]
It's a long time since I looked at TextMate grammars. I'll trust that you have smoke tested this?
We also don't seem to have tests in this repo. 😱

**Comment by Jonathan-Casey** [comment]
@mttrbrts yes, it passed the eye test, via the chrome script.

**Pull request merged**